### DISCUSSION TITLE
Ionic React ou React Native

### DISCUSSION THREAD

**Discussion by wilsonneto-swe** [frontendbr/forum#1609] [Perguntas e Respostas]
Galera, a discussão aqui que quero fomentar é, em qual situação usar um e em qual usar o outro?

Pensando em desenvolver cross platform vejo os dois quase que com o mesmo objetivo e sentido, o que vcs acham a respeito? Qual situação um é melhor que o outro?

Abraços!

**Comment by kivervinicius** [comment]
Ionic sempre será WebView, React Native roda componentes de forma nativa. Vejo o Ionic como um excelente PWA nada mais que isso.

**Comment by felipefialho** [comment]
#1179 #1573

**Comment by vhfmag** [comment]
- Ionic permitiria usar um código só pra diversos métodos de distribuição (Web, apps e electron), mas se paga o preço em performance devido às limitações da Web. Além disso, o Capacitor é novo, então a estabilidade é incerta;
- React Native dá acesso a uma performance nativa com o preço de compartilhar código com a Web ser custoso (existe o https://npm.im/react-native-web, mas continua existindo um custo não trivial). Além disso, tem um melhor tooling pra code push (até onde sei).

Pra você estar considerando React Native, imagino que só vá fazer apps Android e iOs, então a vantagem do Ionic faz menos sentido...

Eu provavelmente apostaria no Ionic pela simplicidade da solução e por eu ter mais conhecimento em Web, mas cabe pesar as prioridades e fortalezas do seu projeto e da sua equipe. :blush:
